Update cache docs (#3505)
apollographql/apollo-android: docs/source/caching/query-watchers.mdx

---
title: Query watchers
---

> Because the normalized cache deduplicates all your data, and if you defined your cache ids, you can the cache it a the source of truth for your UI and be automatically notified when some data changes. For this you can use `ApolloQueryCall.watch`

Reviewer: ```suggestion
Because the normalized cache deduplicates all your data, and if you defined your cache ids, you can use the cache as the source of truth for your UI and be automatically notified when some data changes. For this you can use `ApolloQueryCall.watch`
```
